.gitignore

```diff
@@ -8,9 +8,11 @@ gen/
 xcuserdata/
 
 # Cocoapods
-
 Pods/
 *.xcworkspace
 
 # SPM
 .swiftpm/
+
+# jazzy
+docs/
```

Comment:
What do you think about using `.docs` if it's ignored, in case someone wants to add a `docs` folder? (I guess they could change it then, but I could see someone scratching their head for a while 😅)

Reply:
Sure